src/lib/apimanifest.csproj

```diff
@@ -35,7 +35,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="System.Text.Json" Version="[6.0,9.0)" />
+    <PackageReference Include="System.Text.Json" Version="[6.0,)" />
```

Comment:
```suggestion
    <PackageReference Include="System.Text.Json" Version="[6.0,10.0)" />
```

Reply:
Setting the upper bound does not do anything to change how NuGet's algorithm behaves when making choices -- all it does is introduce a failure when folks violate the upper bound.  This hinders compat because it prevents/discourages folks from using a library that might (_should_!) just work.

We build tooling that helps folks ensure that they build compatible libraries https://learn.microsoft.com/en-us/dotnet/fundamentals/apicompat/package-validation/overview, and for System.Text.Json we run this tooling and have *very* high bar for compatibility since it's also part of .NET and is updated in place for customers.  

Compatibility is the norm and should be expected.  Only when a known compatibility problem exists should you consider using upper bounds.